Richtlijnen: 'Voorkom fouten' bijschaven (#835)
nl-design-system/documentatie: docs/richtlijnen/formulieren/_help-show-required-intro.md

<!-- @license CC0-1.0 -->

## Vermeld duidelijk of een veld wel of niet verplicht is

> Laat bezoekers duidelijk weten welke informatie ingevuld moeten worden om een formulier te verzenden. Dit kan kan op twee manieren, door de **verplichte** velden te markeren of door de **niet-verplichte** velden te markeren. We geven je hieronder twee opties, waarvan we de eerste optie 'Geef aan welke velden niet verplicht zijn' aanbevelen.

Reviewer: Tot nu toe spreken we vooral over 'gebruikers'.

```suggestion
Laat gebruikers duidelijk weten welke informatie ingevuld moeten worden om een formulier te verzenden. Dit kan kan op twee manieren.

1. Door de **niet-verplichte** velden te markeren.
2. Door de **verplichte** velden te markeren.

Op basis van gebruikersonderzoek zouden wij de eerste optie aanbevelen.
```

> Plaats extra uitleg over wat al dan niet verplicht is altijd boven het formulier, zodat de gebruiker weet wat te verwachten bij het invullen.

Reviewer: ```suggestion
Plaats de uitleg over wel of niet verplichte velden altijd boven het formulier, zodat de gebruiker weet wat te verwachten bij het invullen.
```

> Zet deze zichtbare informatie altijd in het label en niet er buiten. Screenreadergebruikers krijgen deze informatie voorgelezen door `aria-required` of `required` in de code op te nemen bij de verplichte velden.

Reviewer: ```suggestion
Maak de markering onderdeel van het label. Screenreadergebruikers krijgen deze informatie voorgelezen door `aria-required` of `required` in de code op te nemen bij de verplichte velden.
```

Author: Nu ik het nalees is dit misschien onduidelijk, nu lijkt het net of het bij de screen readers over de info in het label gaat.

Voorstel:
> Maak de markering onderdeel van het label. 
Screenreadergebruikers krijgen deze informatie daarnaast ook voorgelezen door `aria-required` of `required` in de code op te nemen bij de verplichte velden.
